docs/user/lectures.rst

```diff
@@ -39,10 +39,21 @@ Managing Lectures
 Instructors have three options for creating a lecture.
 
     1. Create a new lecture from scratch by clicking on |create-lecture-btn|.
-    Lectures consist of a title, a description, and optionally a start and end date.
+    Lectures consist of a title, a description, optionally a start and end date, and the selection of automatic unit processing.
 
         |create-lecture|
 
+    If instructors decide to process units automatically by clicking on the |process-units-checkbox| checkbox, additional information will be displayed
+    and they can click on |process-units-btn| to process the lecture PDF into multiple units.
+
+       |create-lecture-automatic-unit-processing|
+
+    Once the units are processed, instructors can view how the units are split (start page and end page). A name is automatically generated for each lecture unit. Instructors
```

Comment:
```suggestion
    Once the units are processed, instructors can view how the units are split, beginning from a start page to an end page. A name is automatically generated for each lecture unit. Instructors
```
I just try to avoid parentheses :)